src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -593,10 +635,16 @@ MulticopterPositionControl::parameters_update(bool force)
 		_params.hold_max_xy = math::max(0.f, v);
 		param_get(_params_handles.hold_max_z, &v);
 		_params.hold_max_z = math::max(0.f, v);
-		param_get(_params_handles.acc_up_max, &v);
-		_params.acc_up_max = v;
-		param_get(_params_handles.acc_down_max, &v);
-		_params.acc_down_max = v;
+		param_get(_params_handles.rc_flt_smp_rate, &(_params.rc_flt_smp_rate));
+		_params.rc_flt_smp_rate = math::max(1.0f, _params.rc_flt_smp_rate);
+		/* since we use filter to detect manual direction change, take half the cutoff of the stick filtering */
+		param_get(_params_handles.rc_flt_cutoff * 0.7f, &(_params.rc_flt_cutoff));
+		/* make sure the filter is in its stable region -> fc < fs/2 */
+		_params.rc_flt_cutoff = math::constrain(_params.rc_flt_cutoff, 0.1f, (_params.rc_flt_smp_rate / 2.0f) - 1.f);
```

Comment:
yes, need to set it as well, but probably it is best to do it within the sensor module. 
anyway, your comments are noted and written down and will add it to a different pr